### PR TITLE
Adds Vagrant config for Fedora 26

### PR DESCRIPTION
**Pull request by HelioGuilherme66**
Inspired by #520 I setup this Fedora 26 vagrant file.
(please adjust memory and cores to your host machine)

Thank you for making this so easy.

### CODE REVIEW

**Comment by RobinD42** [comment]
Are there any issues with using the "fedora/26-atomic-host" box? When possible I'd like to stick with the official vagrant boxes from the distro maintainers.

**Comment by HelioGuilherme66** [comment]
I selected that one because:

> based on Vagrant Cloud fedora/26-cloud-base; with Virtualbox Guest Additions ver 5.1.26; localised to local repo server for Fedora updates

But I'll change to " fedora/26-cloud-base".

**Comment by RobinD42** [comment]
I've already made the change locally along with a few other things. I'll go ahead and merge this and push a few more related commits a little later after some more testing.  Thanks!

File: vagrant/fedora-26/first_time_setup.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This script is to be run after `vagrant up` failing
+# with failure to setup shared folders.
+#
+# It downloads and installs Virtual Box Guest Additions.
```
Comment (RobinD42) [inline vagrant/fedora-26/first_time_setup.sh]: See the README in the parent folder. If you have the vagrant-vbguest plugin installed then the Guest Additions will be installed before the bootstrap.sh script is run during provisioning, and there is no need to do anything extra for it.

Comment (HelioGuilherme66) [reply vagrant/fedora-26/first_time_setup.sh]: I've missed it!

I will now close this PR because you already have it on master.

Thanks again!

**Comment by HelioGuilherme66** [comment]
Master have already a fedora-26 configuration. This PR is no longer needed. Closing.

**Pull request closed without merging**